examples/opf/tools/sp_plotter.py

```diff
@@ -26,7 +26,7 @@
 import copy
 import csv
 import numpy as np
-from nupic.research import FDRCSpatial2
+from nupic.research import SpatialPooler
```

Comment:
from nupic.research.SpatialPooler import SpatialPooler 

so you don't need to do sp = SpatialPooler.SpatialPooler below